lib/eChronicler/journal_entries.ex

```diff
@@ -8,10 +8,20 @@ defmodule EChronicler.JournalEntries do
     Repo.all(JournalEntry)
   end
 
+  def reverse_chron_list_journal_entries() do
+    Repo.all(JournalEntry)
+    |> Enum.sort(&(&1.inserted_at > &2.inserted_at))
+  end
+
   def create_journal_entry(attrs \\ %{}) do
     %JournalEntry{}
     |> JournalEntry.changeset(attrs)
     |> Repo.insert()
   end
 
+  def truncate_journal_entry(entry) do
+    sliced = String.slice(entry, 0..50)
+    sliced <> "..."
+  end
```

Comment:
Both `truncate_journal_entry()` and `create_journal_entry()` are acting on singular entries, so it might make more sense for them to live in `EChronicler.EChronicler.JournalEntry`. Or you could potentially just combine them both into a single module. Up to you!

Reply:
Done! They are both in `EChronicler.EChronicler.JournalEntry` now!